File: test/Automation_scala_CodeGen_Quality_RealRepo_scala/pipelines/Scala_E2E_Existing_Pipeline_All_1/code/src/main/scala/io/prophecy/pipelines/scala_e2e_existing_pipeline_all_1/graph/scalaSG/Subgraph_2/RowDistributor_1.scala
package io.prophecy.pipelines.scala_e2e_existing_pipeline_all_1.graph.scalaSG.Subgraph_2

import io.prophecy.libs._
import io.prophecy.pipelines.scala_e2e_existing_pipeline_all_1.udfs.PipelineInitCode._
import io.prophecy.pipelines.scala_e2e_existing_pipeline_all_1.udfs.UDFs._
import io.prophecy.pipelines.scala_e2e_existing_pipeline_all_1.graph.scalaSG.Subgraph_2.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object RowDistributor_1 {

  def apply(context: Context, in: DataFrame): (DataFrame, DataFrame) =
    (in.filter(col("customer_id") > lit(2)),
     in.filter(col("customer_id") > lit(3))
    )

}

File: test/Automation_python_CodeGen_Quality_RealRepo_python/pipelines/Python_E2E_Existing_Pipeline_All_2/code/python_e2e_existing_pipeline_all_2/graph/pythonSG/Subgraph_2/RowDistributor_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from .config import *
from python_e2e_existing_pipeline_all_2.udfs.UDFs import *

def RowDistributor_1(spark: SparkSession, in0: DataFrame) -> (DataFrame, DataFrame):
    df1 = in0.filter((col("customer_id") > lit(2)))
    df2 = in0.filter((col("customer_id") > lit(4)))

    return df1, df2
